make/source.mk

```diff
@@ -173,6 +173,7 @@ COMMON_SRC = \
             io/gps.c \
             io/ledstrip.c \
             io/pidaudio.c \
+	    locales/translations.c \
```

Comment:
Perhaps `localisation.c` instead of `translations.c`

Reply:
no issue, changed